en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-configuration.md

```diff
@@ -0,0 +1,97 @@
+# Setting up the BigQuery Environment  
+
+The BigQuery connector allows you to access the [BigQuery REST API](https://cloud.google.com/bigquery/docs/reference/rest) through WSO2 through WSO2 EI.
+
+To work with the BigQuery connector, you need to have a Google Cloud Platform account. If you do not have a Google Cloud Platform account, go to [console.cloud.google.com](https://console.cloud.google.com/freetrial), and create a Google Cloud Platform trial account.
+
+BigQuery uses the OAuth 2.0 protocol for authentication and authorization. All requests to the BigQuery REST API must be authorized by an authenticated user. For information on how to obtain authentication and authorization user credentials, you can get the user credentials from the Google Cloud Platfrom using two different manner. See the following section.
```

Comment:
OAuth 2.0 is for authorization, not for authentication.

Reply:
All requests to the BigQuery REST API will be authorized against a registered user. 
Developers can generate user credentials from the Google Cloud Platform using two different mechanisms. 
See the following sections for detail.

Reply:
Fixed.